sdks/go/pkg/beam/testing/passert/passert_test.go

```diff
@@ -0,0 +1,126 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package passert
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/apache/beam/sdks/go/pkg/beam"
+	"github.com/apache/beam/sdks/go/pkg/beam/testing/ptest"
+)
+
+func TestTrue_string(t *testing.T) {
+	p, s := beam.NewPipelineWithRoot()
+	col := beam.Create(s, "a", "a", "a")
+	pred := func(input string) bool {
```

Comment:
While DoFns and PCollections should definitely have variables attached, since the predicate here is small, and it's the final parameter of the True call, it can be defined inline without hurting readability.

Same comment for the cases below.

Reply:
Done.